controllers/comment.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const {ValidationError} = require('sequelize');
+
+const router = express.Router();
+const responseUtil = require('../helpers/response');
+const { Comment, User } = require('../models');
+
+const createComment = (req, res) => {
+    try {
+        const {photo_id,comment} = req.body;
+        const {id} = req.user;
+        Comment.create({photo_id,comment,user_id:id})
+            .then((data) => {
+                responseUtil.successResponse(
+                    res,
+                    `Hi your Comment added`,
+                    {photo: {id: data.id, photo_id, comment, user_id: id, updatedAt: data.updatedAt, createdAt: data.createdAt}}
```

Comment:
key objectnya gunakan `comment`
```suggestion
                    {comment: {id: data.id, photo_id, comment, user_id: id, updatedAt: data.updatedAt, createdAt: data.createdAt}}
```